samples/quickstart.js

```diff
@@ -14,55 +14,249 @@
 
 'use strict';
 
-async function main() {
-  // [START compute_quickstart]
-  const compute = require('@google-cloud/compute');
+// [START compute_instances_list]
+// [START compute_instances_list_all]
+// [START compute_instances_create]
+// [START compute_instances_delete]
+// [START compute_instances_operation_check]
 
-  function sleep(timeoutMs) {
-    return new Promise(resolve => {
-      setTimeout(resolve, timeoutMs);
-    });
-  }
+const compute = require('@google-cloud/compute');
+const compute_protos = compute.protos.google.cloud.compute.v1;
 
-  const client = new compute.AddressesClient({fallback: 'rest'});
-  const project = await client.getProjectId();
-  const region = 'us-east1';
+// [END compute_instances_operation_check]
+// [END compute_instances_delete]
+// [END compute_instances_create]
+// [END compute_instances_list_all]
+// [END compute_instances_list]
 
-  const addressResource = {
-    name: 'test-address-123',
-  };
+// [START compute_instances_list]
```

Comment:
What's the difference between this sample and the existing `gce_list_vms` in https://github.com/googleapis/nodejs-compute/blob/master/samples/listVMs.js

Can we consolidate both examples?

Reply:
So, the new API will be incompatible with current. I can consolidate and update listVMs file.

Reply:
And our goal: make samples in different languages pretty the same. For example listAllVMs in python-compute: https://github.com/googleapis/python-compute/pull/55/files#diff-1f5fe2dfc12cfe0c5f2845b7e628f1ad5e70dcf895f4c3c2a0119e20ddede292R64